coalib/bearlib/naming_conventions/__init__.py

```diff
@@ -130,3 +130,45 @@ def to_spacecase(string):
     return re.sub('(?<=[^\s])([A-Z])',
                   lambda match: ' ' + match.group(1),
                   string)
+
+
+def to_kebabcase(string):
+    """
+    Converts the given string to kebab-case.
+
+    >>> to_kebabcase('HelloWorld')
+    'hello-world'
```

Comment:
In that case I think there should be a standard that we use for coala in particular, and since npm does look to have them all down, maybe we should use it for the other naming conventions aswell?
Open for discussion :)